rstest_macros/src/parse/rstest/files.rs

```diff
@@ -523,10 +580,19 @@ impl ValueListFromFiles<'_> {
             }
 
             let path_str = abs_path.to_string_lossy();
-            values.push((
-                parse_quote! {
+            let value = match refs.files_mode {
+                FilesMode::Path => parse_quote! {
                     <::std::path::PathBuf as std::str::FromStr>::from_str(#path_str).unwrap()
                 },
+                FilesMode::IncludeStr => parse_quote! {
+                    include_str!(#path_str)
+                },
+                FilesMode::IncludeBytes => parse_quote! {
+                    include_bytes!(#path_str)
+                },
+            };
```

Comment:
Ok, now I understood why `include_str!` and `include_bytes!` work... We're using absolute path.... I've forgotten it.